.github/workflows/python-package.yml

```diff
@@ -2,9 +2,9 @@ name: Python package
 
 on:
   push:
-    branches: [ "master" ]
   pull_request:
-    branches: [ "master" ]
+  schedule:
```

Comment:
Could you change this section to what I currently have in the repo, i.e.,
```
on:
  push:
    branches: [ "master" ]
  pull_request:
    branches: [ "master" ]
```

Needed to trigger the test pipelines...

Reply:
I think the pipeline needed triggering because I was opening a PR from a fork without developer approval. I removed these lines because they're [filters](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#using-filters) to only run CI on `master`. My thought was that it would be better to run it on all branches so we know if a PR passes CI before merging in `master`. For instance, I needed to remove these lines to get CI to trigger for me on this PR (e.g., https://github.com/lewisblake/optimParallel-python/actions/runs/7267252816/job/19800655430). I could be wrong though, so I can reintroduce it if you prefer,

Reply:
You are right... looks ok.